src/animations/animatedImage/index.js

```diff
@@ -1,22 +1,30 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import {Animated, View} from 'react-native';
+import {Animated, View, StyleSheet} from 'react-native';
+import {Image} from '../../../src';
```

Comment:
Don't import components from `src` when you are in `src/...` path. 
We already talked about it can create a circular dependency